fireplace/cards/utils.py

```diff
@@ -39,52 +41,40 @@
 
 
 # take a list of weights and a list of card pools and produce a random weighted sample without replacement
```

Comment:
Minor nit: make the comments conform to docstring conventions. Here's a good example:

http://sphinxcontrib-napoleon.readthedocs.org/en/latest/example_google.html